spotifyMini/Models/Analysis.swift

```diff
@@ -7,51 +7,44 @@
 //
 
 import Foundation
+import RealmSwift
 
 let AnalysisFetchedTrackInfoNotification = "AnalysisFetchedTrackInfoNotification"
 let AnalysisNotificationErrorUserInfoKey = "AnalysisUserInfoError"
 
 class Analysis {
     private let spotify = Spotify.manager
     private let tracks: [SPTPartialTrack]
-    private var trackAnalyses: [TrackAnalysis]?
+    private var trackAnalyses = [String : TrackAnalysis]()
```

Comment:
let's append a map in this variable name - this will help us keep tracks, Analysis, and [TrackAnalysis] all straight.

So name is trackAnalysisMap